OpenRA.Mods.Common/Traits/Tooltip.cs

```diff
@@ -21,11 +21,13 @@ public abstract class TooltipInfoBase : ConditionalTraitInfo, Requires<IMouseBou
 	}
 
 	[Desc("Shown in map editor.")]
-	public class EditorOnlyTooltipInfo : TooltipInfoBase
+	public sealed class EditorOnlyTooltipInfo : TooltipInfoBase
 	{
-		public override object Create(ActorInitializer init) { return this; }
+		public override object Create(ActorInitializer init) { return new EditorOnlyTooltip(); }
 	}
 
+	public sealed class EditorOnlyTooltip { }
```

Comment:
is this the only instance this is needed? How did you find this?

Reply:
See the screenshot in the PR description; this is the only one that popped up.